src/pages/applicant/viewingKey.tsx

```diff
@@ -115,9 +152,21 @@ const ViewingKeyPage = () => {
     <div className="px-10 sm:px-14 z-50 mt-20 mb-20 sm:mt-20 md:-mt-8">
       <div className="w-full text-center">
         <div className="z-50 opacity-100 px-0 sm:p-10 flex flex-col">
-          <h2 className="z-50 font-semibold text-2xl sm:text-3xl md:text-3xl lg:text-4xl p-0">
-            {'Create a Viewing Key'}
-          </h2>
+          <div>
```

Comment:
the icon is not clickable and the text is now behind the white lines. You may have to add zindex and/or change the display